packages/commerce-sdk-react/src/hooks/useDNT.ts

```diff
@@ -8,27 +8,34 @@ import useAuthContext from './useAuthContext'
 
 interface useDntReturn {
     dntStatus: boolean | undefined
+    finalDntValue: boolean | undefined
     updateDNT: (preference: boolean | null) => Promise<void>
 }
 
 /**
  * Hook that returns
- * dntStatus - a boolean indicating the current DNT preference
- * updateDNT - a function that takes a DNT preference and creates the dw_dnt
- *              cookie and reauthroizes with SLAS
- *
+ * dntStatus - a boolean indicating the DNT user choice status. Used to determine
```

Comment:
What about `selectedDoNotTrackValue` (for dntStatus) and `effectiveDoNotTrackValue` (for finalDntValue)?